KK_ButtPlugin/ButtplugController.cs

```diff
@@ -82,9 +82,18 @@ public class ButtplugController : UnityEngine.MonoBehaviour
                 { "Lotus (One Leg Up).OLoop", 0.9597445f }
             };
 
-        private readonly ButtplugWsClient client = new ButtplugWsClient();
+        private ButtplugWsClient client = new ButtplugWsClient();

```

Comment:
Why can't this be readonly?

Reply:
sorry, I originally was deleting the client and making a new one instead of just invoking close and open